next.config.js

```diff
@@ -363,23 +371,23 @@ async function redirects() {
     { source: '/references', destination: '/references/blockstack-cli', permanent: true },
     { source: '/ecosystem', destination: '/ecosystem/overview', permanent: true },
     {
-      source: '/stacks-blockchain/testnet-node',
```

Comment:
same as the last comment - let's update the destination here but leave the source as is

Reply:
I added back these redirects and fixed their destination fields.